.gitmodules

```diff
@@ -58,3 +58,6 @@
 [submodule "scripts/packages/IJuliaCore"]
 	path = scripts/packages/IJuliaCore
 	url = https://github.com/JuliaLang/IJuliaCore.jl.git
+[submodule "scripts/packages/Tar"]
+	path = scripts/packages/Tar
+	url = https://github.com/JuliaIO/Tar.jl.git
```

Comment:
The changes in this file don't seem to match the submodules that are added in this PR? Not sure how that can happen...